_posts/2023-06-26-trino-fest-2023-salesforce.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+title: "Trino Fest 2023 recap: Anomaly detection for Salesforce’s production data using Trino"
+author: "Tuli Nivas, Geeta Shankar, and Ryan Duan"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/Salesforce.png
+show_pagenav: false
+---
+
+Rolling into our next presentation of Trinofest 2023, we're excited to bring you
+Tuli Navas and Geeta Shankar's talk from the Performance Engineering Team of
+Salesforce. They provide numerous reasons for why they need Trino and
+further continue explaining how it is essential for anomaly detection within
+their data. It's an insightful talk of how expeditious Trino can be that you definitely
+don't want to miss.
+
+<!--more-->
+
+{% youtube nFuqpb2GjVI %}
+
+## Recap
+
+Salesforce provides customer relationship management software and applications
+focused on sales, customer service, marketing automation, e-commerce, analytics,
+and application development. They host hundreds of thousands of customers that
+generate millions of transactions per day. As Tuli said, "Trino has been a
+one-stop shop for analytics". Trino allows them to do a multitude
+of things such as, querying all the data quickly and efficiently, and making it
+accessible in an optimal, easy, and timely fashion.
```

Comment:
```suggestion
one-stop shop for analytics". Trino allows them to do everything they need.
```
Let's keep it simple here, we've already made it pretty clear that quickly querying data is what Trino's all about.